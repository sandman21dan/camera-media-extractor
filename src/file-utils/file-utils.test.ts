import { findRegularFilesInDir } from ".";
import { resolve } from "path";

it('finds all regular in a given directory files', (done) => {
  const dirName = resolve(__dirname, 'test');

  findRegularFilesInDir(dirName).then((files) => {
    expect(files.length).toEqual(20);
    expect(files).toMatchSnapshot();
    done();
  });
});
