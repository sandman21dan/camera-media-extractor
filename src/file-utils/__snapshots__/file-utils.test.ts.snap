// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`finds all regular in a given directory files 1`] = `
Array [
  "/mnt/c/Users/danie/Documents/development/camera-media-extractor/src/file-utils/test/file_1.txt",
  "/mnt/c/Users/danie/Documents/development/camera-media-extractor/src/file-utils/test/file_10.txt",
  "/mnt/c/Users/danie/Documents/development/camera-media-extractor/src/file-utils/test/file_15.txt",
  "/mnt/c/Users/danie/Documents/development/camera-media-extractor/src/file-utils/test/file_17.txt",
  "/mnt/c/Users/danie/Documents/development/camera-media-extractor/src/file-utils/test/file_19.txt",
  "/mnt/c/Users/danie/Documents/development/camera-media-extractor/src/file-utils/test/file_2.txt",
  "/mnt/c/Users/danie/Documents/development/camera-media-extractor/src/file-utils/test/file_20.txt",
  "/mnt/c/Users/danie/Documents/development/camera-media-extractor/src/file-utils/test/file_3.groovy",
  "/mnt/c/Users/danie/Documents/development/camera-media-extractor/src/file-utils/test/file_4.txt",
  "/mnt/c/Users/danie/Documents/development/camera-media-extractor/src/file-utils/test/file_5.txt",
  "/mnt/c/Users/danie/Documents/development/camera-media-extractor/src/file-utils/test/file_6.txt",
  "/mnt/c/Users/danie/Documents/development/camera-media-extractor/src/file-utils/test/file_7.txt",
  "/mnt/c/Users/danie/Documents/development/camera-media-extractor/src/file-utils/test/file_8.txt",
  "/mnt/c/Users/danie/Documents/development/camera-media-extractor/src/file-utils/test/file_9.txt",
  "/mnt/c/Users/danie/Documents/development/camera-media-extractor/src/file-utils/test/folder1/file_13.txt",
  "/mnt/c/Users/danie/Documents/development/camera-media-extractor/src/file-utils/test/folder1/file_14.txt",
  "/mnt/c/Users/danie/Documents/development/camera-media-extractor/src/file-utils/test/folder1/file_16.txt",
  "/mnt/c/Users/danie/Documents/development/camera-media-extractor/src/file-utils/test/folder2/file_11.txt",
  "/mnt/c/Users/danie/Documents/development/camera-media-extractor/src/file-utils/test/folder2/file_12.txt",
  "/mnt/c/Users/danie/Documents/development/camera-media-extractor/src/file-utils/test/folder2/file_18.txt",
]
`;
